resources/assets/less/bem/forum-post-info.less

```diff
@@ -75,6 +75,9 @@
     }
 
     &--username {
+      width: 100%;
+      display: block;
```

Comment:
this is not supposed to show up on mobile view

Reply:
oops, you're right
i think removing the display: block solves this issue
```suggestion
```